service/user/service.go

```diff
@@ -0,0 +1,120 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/stasBigunenko/monorepa/pkg/storage/newStorage"
+	"unicode"
+
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+)
+
+type UsrService struct {
+	Us      model.UserHTTP
+	storage newStorage.NewStore
+}
+
+func NewUsrService(s newStorage.NewStore) *UsrService {
+	return &UsrService{
+		storage: s,
+	}
+}
+
+func (u *UsrService) Get(_ context.Context, id uuid.UUID) (model.UserHTTP, error) {
+
+	res, err := u.storage.Get(context.Background(), id)
+	if res == nil && err == nil || err != nil {
+		return model.UserHTTP{}, errors.New("not found")
+	}
+
+	err = json.Unmarshal(res, &u.Us)
+	if err != nil {
+		return model.UserHTTP{}, errors.New("unmarshal problem")
+	}
+
+	return u.Us, nil
+}
+
+func (u *UsrService) GetAll(_ context.Context) ([]model.UserHTTP, error) {
+
+	res, err := u.storage.GetAll(context.Background())
+	if err != nil {
+		return nil, errors.New("not found")
+	}
+
+	ac := []model.UserHTTP{}
+
+	for _, val := range res {
+		err := json.Unmarshal(val, &u.Us)
+		if err != nil {
+			return nil, errors.New("unmarshal problem")
+		}
+
+		ac = append(ac, u.Us)
+	}
+	return ac, nil
+}
+
+func (u *UsrService) Create(_ context.Context, b string) (model.UserHTTP, error) {
```

Comment:
Can update value names(b is strange choose)

Reply:
Not important)

Reply:
updated value to "name"